jsk_2016_01_baxter_apc/launch/include/segmentation_for_each_hand.launch

```diff
@@ -62,11 +62,18 @@
               to: $(arg INPUT_CLOUD)
             - from: ~indices
               to: $(arg HAND)_sib_mask_image_to_pi/output
-        - name: $(arg HAND)_sib_centroid_publisher
-          type: jsk_pcl/CentroidPublisher
+        - name: $(arg HAND)_sib_pc_to_cpi
+          type: jsk_pcl_utils/PointCloudToClusterPointIndices
           remappings:
-            - from: ~input
-              to: $(arg HAND)_sib_extract_pi/output
+          - from: ~input
+            to: $(arg HAND)_sib_extract_pi/output
+        - name: $(arg HAND)_sib_cpi_decomposer
+          type: jsk_pcl/ClusterPointIndicesDecomposer
+          remappings:
+          - from: ~input
+            to: $(arg HAND)_sib_extract_pi/output
+          - from: ~target
+            to: $(arg HAND)_sib_pc_to_cpi/output
```

Comment:
Indentation in `remappings` is different between before line 64 and after.
Which is correct?